beacon-chain/db/db_metrics.go

```diff
@@ -0,0 +1,70 @@
+package db
+
+import (
+	"github.com/boltdb/bolt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	dbPageSize = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "page_size",
```

Comment:
Please prefix all of these with beaconchain_db_

We need to update all metrics to start with the application name, then specific metric.